modules/scalacheck/src/weaver/scalacheck/Checkers.scala

```diff
@@ -22,6 +22,54 @@ trait Checkers {
   // Configuration for property-based tests
   def checkConfig: CheckConfig = CheckConfig.default
 
+  // Implicit defs for ioPropF
+  implicit def ioPropF[A]: PropF[F[A]] = new Prop[F, F[A]] {
```

Comment:
I gave this some thoughts, and I don't think I'm confortable with this. It was a conscious decision to force the user to return expectations in their tests, in order to encourage them to separate the test logic from the expectations, and not intertwine them much. Allowing them to pass an arbitrary `F[A]` in `forall` calls would break cohesion with the rest of the library, and that's not really desirable. Especially because it's not the end of the world for people to add it in userland, exactly like you've done ... It's literally 10 lines of code. 

That being said, I can empathise with the fact that it's a annoying that there's not any short-hand way to turn an `F[A]` into an `F[Expectations]`. 

Maybe a better way would be to add an extension method that would allow for it. The code could look like : 

```scala  
forall(gen) { x  => 
   f(x).expectSuccess 
}
```

Reply:
I that case I think, should close the PR. I did get better insights into the code. So thanks for that.